src/actions/SearchActions.ts

```diff
@@ -9,7 +9,7 @@ export const SEARCH = {
 /*
     action creators
 */
-export const loadSearchCards = (cards) => {
+export const loadSearchCards = (cards: any) => {
```

Comment:
Lets refrain from using any to the best of our understanding with typescript

Reply:
Yes, we should scarcely use the type "any".

Reply:
no need to repeat what I've just said